assets/reviews.js

```diff
@@ -1,89 +1,128 @@
+/**
+ * Converts date from yyyy-mm-dd to dd.mm.yyyy
+ */
+function convertDate() {
+  const createdAtDate = document.querySelectorAll('.created-at-date');
+
+  if (createdAtDate) {
+    createdAtDate.forEach(date => {
+      const originalDateString = date.textContent;
+      const originalDate = new Date(originalDateString);
+      const day = originalDate.getDate().toString().padStart(2, '0');
+      const month = (originalDate.getMonth() + 1).toString().padStart(2, '0');
+      const year = originalDate.getFullYear().toString();
+      const formattedDate = `${day}.${month}.${year}`;
+      date.textContent = formattedDate;
+    });
+  };
+}
+
 (async () => {
   const reviewsContainer = $('.yc-product-reviews');
   const reviewsWrapper = $('.yc-reviews-wrapper');
   const showMoreButton = $('#show-more');
   let reviews = [];
 
-    const noDataSetter = (element) => {
-      if (reviewsContainer) {
-        reviewsContainer.remove();
-      }
-    };
+  /**
+   * This function is used to remove the reviews container if there is no data.
+   * 
+   * @param {HTMLElement} element - The element you want to target
+   */
+  const noDataSetter = () => {
```

Comment:
It would be preferable if you could give it a more meaningful name, such as 'removeReviews.'